File: HelperJobby/client-app/src/DTOs/resumeRelatedDTOs/ResumeDTO.ts
import {JobSeekerDTO} from "../accountDTOs/JobSeekerDTO";
import {EducationDTO} from "./EducationDTO";
import {WorkExperienceDTO} from "./WorkExperienceDTO";
import {SkillDTO} from "./SkillDTO";

export interface ResumeDTO {
    id: number;
    jobSeekerId: number;
    createdOn : string;
    jobSeeker: JobSeekerDTO;
    educations: EducationDTO[];
    workExperiences: WorkExperienceDTO[];
    skills: SkillDTO[];
}

File: HelperJobby/DTOs/Resume/ResumeDTO.cs
using HelperJobby.DTOs.Account;

namespace HelperJobby.DTOs.Resume;

public class ResumeDTO
{
    public int Id { get; set; }

    public int JobSeekerId { get; set; }
    public DateOnly CreatedOn { get; set; }

    public JobSeekerDTO JobSeeker { get; set; }

    public List<EducationDTO> Educations { get; set; }
    public List<WorkExperienceDTO> WorkExperiences { get; set; }
    public List<SkillDTO> Skills { get; set; }
}